src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CountDocumentsCommandResolver.java

```diff
@@ -0,0 +1,40 @@
+package io.stargate.sgv2.jsonapi.service.resolver.model.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.impl.CountDocumentsCommands;
+import io.stargate.sgv2.jsonapi.service.operation.model.Operation;
+import io.stargate.sgv2.jsonapi.service.operation.model.ReadType;
+import io.stargate.sgv2.jsonapi.service.resolver.model.CommandResolver;
+import io.stargate.sgv2.jsonapi.service.resolver.model.impl.matcher.FilterableResolver;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+/** Resolves the {@link CountDocumentsCommands } */
+@ApplicationScoped
+public class CountDocumentsCommandResolver extends FilterableResolver<CountDocumentsCommands>
+    implements CommandResolver<CountDocumentsCommands> {
+  @Inject
+  public CountDocumentsCommandResolver(ObjectMapper objectMapper) {
+    super(objectMapper);
+  }
+
+  public CountDocumentsCommandResolver() {
+    this(null);
```

Comment:
Is this needed for testing or ... ? (guessing so if passing `null` `ObjectMapper` is ok)

Reply:
Not required, removing it.